CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ CHANGELOG for Sulu
 
 * dev-develop
     * BUGFIX      #1191 [AdminBundle]    Fixed unique Navigation ID
+    * ENHANCEMENT #1342 [SecurityBundle] Added fullContact group to creator and changer property of contact
```

Comment:
Added creator and changer of contact to 'fullContact' serialization group

would probably be clearer